Since Qt5-5.5.0 explicit include <QIODevice> is required
if QIODevice is used in headers or source files. It seems
it was included as a side-effect by other header files before.

--- src/qca_publickey.cpp	2014-11-06 09:15:45.000000000 +0100
+++ src/qca_publickey.cpp	2015-07-07 09:26:59.819005743 +0200
@@ -25,6 +25,7 @@
 
 #include <QFile>
 #include <QTextStream>
+#include <QIODevice>
 
 namespace QCA {
 
--- src/qca_keystore.cpp	2014-11-06 09:15:45.000000000 +0100
+++ src/qca_keystore.cpp	2015-07-07 09:29:24.886016202 +0200
@@ -27,6 +27,7 @@
 #include <QSet>
 #include <QMutex>
 #include <QWaitCondition>
+#include <QIODevice>
 
 #include <stdlib.h> // abort
 #include <stdio.h>  // fprintf
--- include/QtCrypto/qca_core.h	2014-11-06 09:15:45.000000000 +0100
+++ include/QtCrypto/qca_core.h	2015-07-07 09:36:30.737046903 +0200
@@ -38,6 +38,7 @@
 #include <QList>
 #include <QSharedData>
 #include <QSharedDataPointer>
+#include <QIODevice>
 #include "qca_export.h"
 #include "qca_support.h"
 #include "qca_tools.h"
